pvlib/pvsystem.py

```diff
@@ -1887,6 +1887,82 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
+def pvsyst_celltemp(
+    poa_global,
+    wind_speed,
+    temp_air,
+    eta_m=0.1,
+    alpha_absorption=0.9,
+    temp_model="freestanding",
+):
+    """
+    Calculate PVSyst cell temperature.
+
+    Parameters
+    ------------
+    poa_global : float or Series
+        Total incident irradiance in W/m^2.
+
+    wind_speed : float or Series
+        Wind speed in m/s at a height of 10 meters.
+
+    temp_air : float or Series
+        Ambient dry bulb temperature in degrees C.
+
+    eta_m : numeric
+        PV module efficiency as a fraction
+
+    alpha_absorption : float
+        Absorption coefficient, default is 0.9
+
+    temp_model : string, list, or dict, default 'freestanding'
+        Model to be used.
+
+        If string, can be:
+
+            * 'freestanding' (default)
+            * 'insulated'
+
+        If tuple/list, supply parameters in the following order:
+
+            * natural_convenction_coeff : float
+                Natural convection coefficient. Freestanding default is 29,
+                fully insulated arrays is 15. Recommended values are between
+                23.5 and 26.5.
+
+            * forced_convection_coeff : float
+                Forced convection coefficient, default is 0. Recommended values
+                are between 6.25 and 7.68.
```

Comment:
I don't know much about the PVsyst model, but it seems strange that we'd specify defaults outside of the recommended values.

Reply:
Same here. My intention was to provide functionally familiar defaults to PVsyst users via the "temp_models" options while trying to give guidance to others manually providing their own coefficients per the insight provided on the [website](https://pvpmc.sandia.gov/modeling-steps/2-dc-module-iv/module-temperature/faiman-module-temperature-model/).

Two options:
- Call them something other than "Recommended values" (e.g. physically valid values)
- Remove recommended ranges entirely and leave it to the user to ensure validity.

Reply:
@cwhanse, what do you think about these defaults?

Reply:
29 / 0 are the PVsyst defaults.  Maybe keep them for that reason.  I'd drop the "Recommended values..." comments because those are from the Faiman paper, not from PVsyst.